extensions/amp-analytics/0.1/config.js

```diff
@@ -252,7 +252,8 @@ export class AnalyticsConfig {
     mergeObjects(expandConfigRequest(defaultConfig), config);
     mergeObjects(expandConfigRequest(this.getTypeConfig_()), config,
         /* predefined */ true);
-    mergeObjects(expandConfigRequest(rewrittenConfig), config);
+    mergeObjects(expandConfigRequest(rewrittenConfig), config,
```

Comment:
so this will also allow `optout` to be defined in configRewriter right?

I agree this is the right way to go. But just to double confirm with  @avimehta @zikas

Reply:
That is correct.

Reply:
I think that is okay.